diff --git a/english/game/editor.msg b/english/game/editor.msg
index 0c291f14..cfbcbcc2 100644
--- a/english/game/editor.msg
+++ b/english/game/editor.msg
@@ -95,18 +95,18 @@
 #
 # Abbreviated Derived/Non Derived stat names
 #
-{300}{}{Hit Points}
+{300}{editor}{Hit Points}              # spanish: line too long, separated
 {301}{}{Action Points }
-{302}{}{Armor Class}
+{302}{editor}{Armor Class}             # spanish: line too long, separated
 {303}{}{Unarmed Damage}
 {304}{}{Melee Damage}
 {305}{}{Damage Res.  }
 {306}{}{Poison Res.   }
 {307}{}{Radiation Res.}
-{308}{}{Sequence}
-{309}{}{Healing Rate}
-{310}{}{Critical Chance}
-{311}{}{Carry Weight}
+{308}{editor}{Sequence}                # spanish: line too long, separated
+{309}{editor}{Healing Rate}            # spanish: line too long, separated
+{310}{editor}{Critical Chance}         # spanish: line too long, separated
+{311}{editor}{Carry Weight}            # spanish: line too long, separated
 {312}{}{Poisoned}
 {313}{}{Radiated}
 {314}{}{Eye Damage}
diff --git a/data/text/english/game/item.msg b/data/text/english/game/item.msg
index 57da2478..bdd3cb1a 100644
--- a/data/text/english/game/item.msg
+++ b/data/text/english/game/item.msg
@@ -25,7 +25,7 @@
 {600}{}{%s succumbs to the adverse effects of drugs.}
 
 # Weapon Damage strings
-{800}{}{Light}
+{800}{Weapon Damage}{Light} # spanish: separate "light damage" from "light source", ref https://forums.bgforge.net/viewtopic.php?f=14&p=925
 {801}{}{Medium}
 {802}{}{Heavy}
 {803}{}{Extreme}
